readme.md

```diff
@@ -457,6 +457,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
+- [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows) - Workflows for Alfred, the productivity application for Mac OS X that boosts your efficiency with hotkeys, keywords and text expansion.
```

Comment:
This is not succinct. Succinct means **short** and clear.

It's also not Mac OS X, but macOS.